projects/com.oracle.truffle.llvm.pipe.native/Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: default
+
+TARGET=libpipe.so
+
+NATIVE_CLASSES=com.oracle.truffle.llvm.pipe.CaptureOutput
+
+OBJECTS=capture.o
+
+default: bin/${TARGET}
+
+src_gen/native.h: Makefile
+	@echo JAVAH $@
+	@rm -f $@
+	@${JAVA_HOME}/bin/javah -classpath ${MX_CLASSPATH} -o $@ ${NATIVE_CLASSES}
+
+bin/src-gen-deps: Makefile
+	@rm -f $@
+	@mkdir -p bin/
+	@${JAVA_HOME}/bin/javap -sysinfo -classpath ${MX_CLASSPATH} ${NATIVE_CLASSES} | grep "Classfile " | sed -e 's|Classfile |src_gen/native.h: |' >> $@
+
+OBJECTFILES=${OBJECTS:%.o=bin/%.o}
+CFLAGS=-g -fPIC -I${VPATH}/include -Isrc_gen -I${JAVA_HOME}/include -I${JAVA_HOME}/include/darwin $(shell pkg-config)
+LIBS=-g $(shell pkg-config)
+
+bin/${TARGET}: ${OBJECTFILES}
+	@echo LINK $@
+	@g++ -std=gnu++0x -shared ${LIBS} $+ -o $@
```

Comment:
I am not familiar with Makefiles, but should we use `CXX` here to note that we can use any installed C++ compiler? \cc @rschatz